asset-managment/niapm.yml

```diff
@@ -0,0 +1,1405 @@
+      serialNumber:
+        description: Serial number of the asset.
+        type: string
+        example: "01BB877A"
+      modelName:
+        description: Model name of the asset.
+        type: string
+        example: "NI PXIe-6368"
+      modelNumber:
+        description: Model number of the asset.
+        type: integer
+        example : 123
+      vendorName:
+        description: Vendor name of the asset.
+        type: string
+        example: National Instruments
+      vendorNumber:
+        description: Vendor number of the asset.
+        type: integer
+        example: 4244
+      name:
+        description: Name of the asset.
+        type: string
+        example: PCISlot2
+      firmwareVersion:
+        description: Firmware version of the asset.
+        type: string
+        example: A1
+      hardwareVersion:
+        description: Hardware version of the asset.
+        type: string
+        example: 12A
+      busType:
+        $ref: '#/definitions/BusType'
+      visaResourceName:
+        description: VISA resource name of the asset.
+        type: string
+        example : vs-1234
+      assetLocation:
+        $ref: '#/definitions/AssetLocation'
+      temperatureSensors:
+        description: An array of temperature sensor information.
+        type: array
+        items:
+          $ref: '#/definitions/TemperatureSensor'
+      supportsSelfCalibration:
+        description: Specifies whether the asset supports internal (self) calibration.
+        type: boolean
+        example: true
+      supportsExternalCalibration:
+        description: Specifies whether the asset supports external calibration.
+        type: boolean
+        example: true
+      selfCalibration:
+        $ref: '#/definitions/SelfCalibrationModel'
+      externalCalibration:
+        $ref: '#/definitions/ExternalCalibrationModel'
+      calibrationStatus:
+        $ref: '#/definitions/CalibrationStatus'
+      lastUpdatedTimestamp:
+        description: ISO-8601 formatted timestamp specifying the last date that the asset has had a property update.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      isNIAsset:
+        description: Specifies whether this asset is an NI asset (true) or a 3rd party asset (false).
+        type: boolean
+        example: true
+      properties:
+        description: Specifies a set of properties associated with an asset.
+        type: object
+        additionalProperties:
+          type: string
+        example:
+          Key1: Value1
+      keywords:
+        description: Specifies a set of keywords associated with an asset.
+        type: array
+        items:
+          type: string
+          example: Keyword1
+  AssetWithAvailabilityHistoryModel:
+    title: Asset With Availability History
+    description: Contains asset availability history which consists in availability information for each system in which the asset was present.
```

Comment:
"Information about the availability, or connection, history of an asset, which also contains information about each system the asset was present."?

Reply:
Yes, that is more proper.